webapp/src/components/Signup/SignupLifeBank.js

```diff
@@ -140,6 +140,18 @@ const SignupLifeBank = ({
           className={classes.textField}
           onChange={(event) => setField('name', event.target.value)}
         />
+        {
+          // Front-End Connection Example
+          /*
+          <TextField
+            id="name"
+            label={'categories'}
+            variant="outlined"
+            fullWidth
+            className={classes.textField}
+            onChange={(event) => setField('categories', ["Coupons", "Discount", "Gift"])}
+          /> */
+        }
```

Comment:
Check this comments

Reply:
It's made on purpose if you want to test it, if everything is ok i will commit the removal of those lines in order to merge

Reply:
I supposed that but I think we could remove those lines if we want to merge it to master